cprover_bindings/src/irep/to_irep.rs

```diff
@@ -695,6 +717,24 @@ impl ToIrep for Type {
                     (IrepId::CCType, Irep::just_id(IrepId::Float)),
                 ],
             },
+            Type::Float16 => Irep {
+                id: IrepId::Floatbv,
+                sub: vec![],
+                named_sub: linear_map![
+                    (IrepId::F, Irep::just_int_id(10)),
+                    (IrepId::Width, Irep::just_int_id(16)),
+                    (IrepId::CCType, Irep::just_id(IrepId::Float16)),
+                ],
+            },
+            Type::Float128 => Irep {
+                id: IrepId::Floatbv,
+                sub: vec![],
+                named_sub: linear_map![
+                    (IrepId::F, Irep::just_int_id(112)),
```

Comment:
What does 112 represent?

Reply:
Added a comment to clarify, thanks for pointing it out.